packs/minecraft_java_edition/templates/pack.nomad.tpl

```diff
@@ -0,0 +1,135 @@
+[[/* remove `rcon` from `$ports` if `.my.app_enable_rcon` is false */]]
+[[- $ports := .my.ports ]]
+[[- if (ne .my.app_enable_rcon true) ]]
+[[ unset $ports "rcon" ]]
+[[- end ]]
+
+# see https://developer.hashicorp.com/nomad/docs/job-specification/job
+job "[[ .my.job_name ]]" {
+  region      = "[[ .my.region ]]"
+  datacenters = [[ .my.datacenters | toJson ]]
+  type        = "service"
+  namespace   = "[[ .my.namespace ]]"
+  priority    = [[ .my.priority ]]
+
+  # see https://developer.hashicorp.com/nomad/docs/job-specification/group
+  group "[[ .my.group_name ]]" {
+    count = [[ .my.count ]]
+
+    # see https://developer.hashicorp.com/nomad/docs/job-specification/ephemeral_disk
+    ephemeral_disk {
+      migrate = [[ .my.ephemeral_disk.migrate ]]
+      size    = [[ .my.ephemeral_disk.size ]]
+      sticky  = [[ .my.ephemeral_disk.sticky ]]
+    }
+
+    # see https://developer.hashicorp.com/nomad/docs/job-specification/network
+    network {
+      # see https://developer.hashicorp.com/nomad/docs/job-specification/network#network-modes
+      mode = "[[ .my.network_mode ]]"
+
+      [[/* iterate over `$ports` to create Port Mappings */]]
+      [[- range $name, $config := $ports ]]
+      port "[[ $name ]]" {
+        static = [[ $config.port ]]
+        to     = [[ $config.port ]]
+      }
+      [[ end ]]
+    }
+
+    [[- $job_tags := .my.job_tags -]]
+    [[- $service_name := .my.service_name_prefix -]]
+    [[- $service_provider := .my.service_provider -]]
+    [[/* iterate over `$ports` to map Services */]]
+    [[ range $name, $port := $ports ]]
+    # see https://developer.hashicorp.com/nomad/docs/job-specification/service
+    service {
+      name     = "[[ $service_name | replace "_" "-" | trunc 63 ]]-[[ $name ]]"
+      tags     = [[ $job_tags | toJson ]]
+      port     = [[ $port.port ]]
+      provider = "[[ $service_provider ]]"
+
+      # see https://developer.hashicorp.com/nomad/docs/job-specification/check
+      check {
+        name     = [[ $name | quote ]]
+        type     = [[ $port.type | quote ]]
+        [[- if eq $port.type "http" ]]
+        path     = [[ $port.path | quote ]]
+        [[- end ]]
+        interval = "30s"
+        timeout  = "15s"
```

Comment:
Those probably should to be configurable

Reply:
Agreed!